### PR TITLE
Release notes for 143

### PR DESCRIPTION
**Pull request by martinpitt**
The release is still running, so please don't merge yet. But review/corrections appreciated!

### CODE REVIEW

File: content/cockpit-143.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 143.
+
+### "Software Updates" page for package based operating systems
+
+There is a new "Software Updates" page for classic dnf/yum/apt based systems,
+to complement the already existing one for OSTree on Atomic. This uses the
```
Comment (stefwalter) [inline content/cockpit-143.md]: We should add a link to projectatomic.io for Atomic.

File: content/cockpit-143.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+[PackageKit API](https://www.freedesktop.org/software/PackageKit/)
+for platform independence. This feature is shipped in the new
+`cockpit-packagekit` rpm/deb package.
```
Comment (stefwalter) [inline content/cockpit-143.md]: 's/rpm\/deb package/sub-package/'

File: content/cockpit-143.md (hunk 1 above)
Comment (martinpitt) [inline content/cockpit-143.md]: Done.

File: content/cockpit-143.md (hunk 2 above)
Comment (martinpitt) [inline content/cockpit-143.md]: Fixed.

File: content/cockpit-143.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Kubernetes RBAC authorization support
+
+The Kubernetes page can now be used to log into Kubernetes installations that
+use [RBAC authorization](https://kubernetes.io/docs/admin/authorization/rbac/).
+
+### RPM packages for IBM Z systems
+
+The `cockpit-docker` RPM package is now being built on the s390x (IBM Z
+Systems) architecture, as there is a working Fedora Docker package now.
```
Comment (dperpeet) [inline content/cockpit-143.md]: If I were to nitpick, which you asked me to do, I would point out the relatively high local density of `now`s.

Comment (martinpitt) [inline content/cockpit-143.md]: Good point. How about ... ", as the Fedora Docker package has been verified to work
there." instead?

Comment (dperpeet) [inline content/cockpit-143.md]: ... `as the Fedora Docker package is available there.`

Comment (martinpitt) [inline content/cockpit-143.md]: Updated to "as the Fedora Docker package is available there.", thanks for the suggestion.

**Comment by martinpitt** [comment]
Pushed again to add `Closes #` for simple merging.

**Comment by martinpitt** [comment]
Release finished, so let's publish.

**Pull request merged**